pkgs/development/python-modules/python-ndn/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, python3Packages
+, pythonOlder
+, setuptools
+, pycryptodomex
+, lark
+, pygtrie
+, aenum
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-ndn";
+  version = "0.3-1";
+
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "named-data";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-+kWhUNu65yQMHP4m/86j0XqrZGrvtkYOLeLKXjzRhpY=";
+  };
+
+  disabled = pythonOlder "3.9";
+
+  propagatedBuildInputs = [
+    pycryptodomex
+    lark
+    pygtrie
+    aenum
+    setuptools
```

Comment:
setuptools should go to nativeBuildInputs

Reply:
`nativeBuildInputs` will change with the next update  as upstream uses [`poetry-core`](https://github.com/named-data/python-ndn/blob/master/pyproject.toml#L2) in `master`.